core-feature-pack/common/src/main/resources/content/bin/systemd/launch.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
I would suggest /bin/bash so that we can use bash specific mechanism. It also better because with /bin/sh we should test the script with all sort of shell interpreter.

Reply:
We use /sh to maximize the portability now and in the future, it is the same shell used by the launch scripts. Using bash opens the door to not be compatible in the future.

Reply:
Yeah, but do you ensure that CI test the script with different flavor of shell? Unlikely... By using /sh you are opening the door to people filling issue because it does not work with "not Bash". It's generally consider best practises to use explicit /bin/bash for this very reason. As this is a new script, I would take the chance to reduce the scope of incoming issue. That being said, it's not a big deal.

Reply:
Somehow, I can't close this as resolved, but please do!